app/services/post_status_service.rb

```diff
@@ -28,7 +28,7 @@ def call(account, text, in_reply_to = nil, **options)
       status = account.statuses.create!(text: text,
                                         media_attachments: media || [],
                                         thread: in_reply_to,
-                                        sensitive: options[:sensitive],
+                                        sensitive: (options[:sensitive].nil? ? account.user&.setting_default_privacy : options[:sensitive]),
```

Comment:
Why `setting_default_privacy`? That has the values "public", "unlisted", "private", etc...

Reply:
Sorry, distraction on my part, fixing